### PR TITLE
Role group mgmt

### PR DESCRIPTION
**Pull request by harture**

### CODE REVIEW

File: configs/authorization.json [hunk 1]
```diff
@@ -435,6 +513,16 @@
           "end_user": {}
         }
       },
+      "GetRolesOfUser": {
```
Comment (fperot74) [inline configs/authorization.json]: "GetRealmRolesForUser" not removed but not present in this file... Was it missing by error ?

File: pkg/management/endpoint.go [hunk 2]
```diff
@@ -21,6 +21,8 @@ type Endpoints struct {
 	UpdateUser                     endpoint.Endpoint
 	GetUsers                       endpoint.Endpoint
 	CreateUser                     endpoint.Endpoint
+	GetRolesOfUser                 endpoint.Endpoint
+	GetGroupsOfUser                endpoint.Endpoint
 	GetUserAccountStatus           endpoint.Endpoint
 	GetClientRoleForUser           endpoint.Endpoint
 	AddClientRoleToUser            endpoint.Endpoint
```
Comment (fperot74) [inline pkg/management/endpoint.go]: GetRealmRoleForUser endpoint not removed ?

File: configs/authorization.json (hunk 1 above)
Comment (harture) [reply configs/authorization.json]: Should be removed, where I missed it ?

File: pkg/management/endpoint.go (hunk 2 above)
Comment (harture) [reply pkg/management/endpoint.go]: Thanks, fixed

**Pull request merged**